mu-plugins/blocks/language-suggest/language-suggest.php

```diff
@@ -27,3 +27,20 @@ function language_suggest_block_init() {
 	);
 }
 add_action( 'init', __NAMESPACE__ . '\language_suggest_block_init' );
+
+function language_suggest_enqueue_scripts() {
+    wp_enqueue_script(
+        'language-suggest-front',
+        plugins_url( 'src/front.js', __FILE__ ),
+        array(),
+        null,
+        true
+    );
+
+    wp_add_inline_script(
+        'language-suggest-front',
+        'var LanguageSuggestData = ' . json_encode( array( 'locale' => get_locale() ) ) . ';',
```

Comment:
Probably best to use `wp_json_encode()` here just for sanity sake.
```suggestion
        'var LanguageSuggestData = ' . wp_json_encode( array( 'locale' => get_locale() ) ) . ';',
```

JS variables are usually camelCase, but I don't care enough to suggest it to be `languageSuggestData`.

Reply:
Thanks. Should be camelCase, missed that.
`wp_json_encode`: 020a5cd
camelCase: 20258e3